lib/encoders/date_time.ex

```diff
@@ -0,0 +1,5 @@
+defimpl Poison.Encoder, for: Ecto.DateTime do
+  def encode(date_time, options) do
+    date_time |> Ecto.DateTime.to_string |> Poison.encode!(options)
```

Comment:
Also please rename this to `phoenix_ecto/json.ex`. :) I am going to move the existing code to /html.ex soon.